exchanges/binance/binance_websocket.go

```diff
@@ -479,11 +505,69 @@ func (b *Binance) UpdateLocalCache(wsdp *WebsocketDepthStream) error {
 		updateAsk = append(updateAsk, orderbook.Item{Price: p, Amount: a})
 	}
 
-	return b.Websocket.Orderbook.Update(&wsorderbook.WebsocketOrderbookUpdate{
+	return b.Websocket.Orderbook.Update(&buffer.Update{
 		Bids:     updateBid,
 		Asks:     updateAsk,
 		Pair:     currencyPair,
 		UpdateID: wsdp.LastUpdateID,
 		Asset:    asset.Spot,
 	})
 }
+
+// GenerateSubscriptions generates the default subscription set
+func (b *Binance) GenerateSubscriptions() ([]stream.ChannelSubscription, error) {
+	var channels = []string{"@ticker", "@trade", "@kline_1m", "@depth"}
```

Comment:
As discussed on slack 

Although out of scope by default binance websocket updates are provided at 1000ms intervals you can lower this down to 100ms by appending @100ms to the @depth 

I did some testing and found no direct impact to performance in processing additional messages but it does allow us to receive orderbook updates faster